README.md

```diff
@@ -142,16 +159,9 @@ await agent.initialize()
 
 // Resolve dependencies
 const agentContext = agent.dependencyManager.resolve(AgentContext)
-const didResolverService = agent.dependencyManager.resolve(DidResolverService)
```

Comment:
I think this example results the same as the other one. Maybe you can use a generic web-did-resolver instance to illustrate the usage of an external resolver